pkg/curatedpackages/packagecontrollerclient_test.go

```diff
@@ -197,9 +197,9 @@ func TestEnableCuratedPackagesSuccess(t *testing.T) {
 			values = append(values, "cronjob.suspend=true")
 		}
 		tt.chartInstaller.EXPECT().InstallChart(tt.ctx, tt.chart.Name, ociURI, tt.chart.Tag(), tt.kubeConfig, "", valueFilePath, values).Return(nil)
-		any := gomock.Any()
+		gomockAny := gomock.Any()
 		tt.kubectl.EXPECT().
-			GetObject(any, any, any, any, any, any).
+			GetObject(gomockAny, gomockAny, gomockAny, gomockAny, gomockAny, gomockAny).
```

Comment:
We could just stick `gomock.Any()` in here. Not sure this is particularly advantageous.

Reply:
Done.